src/Sulu/Component/Content/StructureManager.php

```diff
@@ -91,6 +91,9 @@ public function setOptions($options)
 
         // overwrite the default values with the given options
         $this->options = array_merge($defaultOptions, $options);
+
+        // FIXME find better solution for default templates in bundle
+        $this->options['structure_paths'] = array_reverse($this->options['structure_paths']);
```

Comment:
i have created a issue for this fixme https://github.com/sulu-cmf/sulu/issues/796